guides/common/modules/con_converting-a-host-to-rhel.adoc

```diff
@@ -0,0 +1,75 @@
+[id="converting-a-host-to-rhel_{context}"]
+= Converting a Host to {RHEL}
+
+You can convert {RHEL} derivative distributions into a supportable {RHEL} on a host while retaining installed applications and configurations.
+{Project} provides the Convert2RHEL utility to simplify the conversion process.
+
+The Convert2RHEL utility in {Project} consists of an Ansible role and an Ansible playbook.
+You use the Ansible role to generate conversion data on {ProjectServer}, that is, repositories, products, activation keys, and host groups.
+Then you perform the actual conversion on the host using the Ansible playbook.
+
+You can convert:
+
+* CentOS Linux 7 to {RHEL} 7
+* Oracle Linux 7 to {RHEL} 7
+* CentOS Linux 8 to {RHEL} 8
+* Oracle Linux 8 to {RHEL} 8
+
+You cannot convert a version 7 system into {RHEL} 8, or a version 8 system into {RHEL} 7.
+
+The conversion process is similar to a minor release upgrade of {RHEL} in which every RPM package on the system is replaced.
+Third-party packages and non-{Team} packages that are not available in {RHEL} are retained.
+
+The Convert2RHEL utility removes unnecessary packages such as logos, or packages known to cause issues during the conversion.
+The utility replaces the `CentOS-release` or `Oracle-release` package with the `rhel-release` package, and all packages signed by CentOS or Oracle with their {Team} equivalents.
+ifdef::satellite[]
+The utility also subscribes the host to {Team} Subscription Management.
+endif::[]
```

Comment:
```suggestion
The utility also subscribes the host to {Team} Subscription Management.
```

I assume this is necessary for all builds; otherwise, RHEL won't receive any content.